external-workloads: do not automatically infer ClusterID/Name (#27886)
cilium-team/cilium: Documentation/operations/upgrade.rst

  labels ``reserved:init``, these policies must be converted to
  ``CiliumClusterwideNetworkPolicy`` by changing the resource type for the
  policy.
* Cluster name and ID are no longer automatically inferred by Cilium agents

>   running on :ref:`external workloads <external_workloads>`, and need to be
>   explicitly specified as parameters (if different from the default values).

Reviewer: Small edits for clarity
```suggestion
  running on :ref:`external workloads <external_workloads>`. If the cluster name
  and ID are different from the default values, you must specify them 
  as parameters.
```
